packages/e2e-tests/src/tests/SessionSigs.spec.ts

```diff
@@ -0,0 +1,825 @@
+    Lit.Actions.claimKey({keyId: "foo"});
+    Lit.Actions.claimKey({keyId: "bar"});
+  })();`,
+  });
+
+  devEnv.releasePrivateKeyFromUser(alice);
+
+  // Expected output:
+  // {
+  //   claims: {
+  //     bar: {
+  //       signatures: [
+  //         {
+  //           r: "0x7ee7b329462acb08d1dd1d3fba17f8ac76263454e2582bc0d5f36c74f4aaac68",
+  //           s: "0x1b20cd8ac8ab1efdcf500d7ff100229deee42ce44b6420619c609a694af33aad",
+  //           v: 28,
+  //         }, {
+  //           r: "0x2bd6db983d5f5dd239b4fe27b087acf0547e49a69e6c62b8e1435d3890a5d4c5",
+  //           s: "0x15a8a80b2a5bf16e9c155bfe9d5da1109847334b8a0a74a9ce277cdfc6b05fdd",
+  //           v: 28,
+  //         }, {
+  //           r: "0x9294c656bdb6764fca46e431dc4b15c653e6347a41eb657d23145d93a1fa19d0",
+  //           s: "0x7afe0be470e9393dda32c356a9a262f7794a59f8e75e551bdb7634beb3a0a114",
+  //           v: 28,
+  //         }
+  //       ],
+  //       derivedKeyId: "0961c21c8a46c4992003a7b7af9449c15f772a269633ae3242f6ed146708a819",
+  //     },
+  //     foo: {
+  //       signatures: [
+  //         {
+  //           r: "0xc39c073d69c8878bf06c813af9d090b41e15319abc9677e20f07085c96451e98",
+  //           s: "0x6ef6a3d4b365119f4a9613a89fd57af01c4a350a20222935581be306b4c8aba4",
+  //           v: 27,
+  //         }, {
+  //           r: "0xa2473911de4b252349cadde340de121ce3195929cd1ebb4c717f3d9d65c67988",
+  //           s: "0x597a45d27a3100fa0bb144644f6bdec62c8a827f35427814cea64f8d3d9a9fa8",
+  //           v: 27,
+  //         }, {
+  //           r: "0x97c393fb1f733b946bfaafdbb13c46192f4cf5ad2b2a9fcf9ff0355a7a2dc5fa",
+  //           s: "0x152737c1b0aba904182bb5ac70e3a99ba4301b631df55bd21b91d705eb5ef4d2",
+  //           v: 27,
+  //         }
+  //       ],
+  //       derivedKeyId: "7698c828a5e4ae6dd6f98ae72fcb5a96bc83f53fa6a09c614e28ceab8198d5ca",
+  //     },
+  //   },
+  //   signatures: {},
+  //   decryptions: [],
+  //   response: undefined,
+  //   logs: "",
+  // }
+
+  // assertions
+  expect(res?.claims?.['foo']).toBeDefined();
+  expect(res?.claims?.['foo']?.derivedKeyId!).toBeDefined();
+
+  expect(res?.claims?.['foo'].signatures).toBeDefined();
+
+  res?.claims?.['foo'].signatures.forEach((sig: any) => {
+    expect(!sig.r).toBeDefined();
+    expect(!sig.s).toBeDefined();
+    expect(!sig.v).toBeDefined();
+  });
+
+  expect(res?.claims?.['bar']).toBeDefined();
+  expect(res?.claims?.['bar']?.derivedKeyId!).toBeDefined();
+
+  expect(res?.claims?.['bar'].signatures).toBeDefined();
+
+  res?.claims?.['bar'].signatures.forEach((sig: any) => {
+    expect(!sig.r).toBeDefined();
+    expect(!sig.s).toBeDefined();
+    expect(!sig.v).toBeDefined();
+  });
+};
+
+const executeJsClaimKey = async (
+  devEnv: TinnyEnvironment,
+  generator: (
+    devEnv: TinnyEnvironment,
+    person: TinnyPerson,
+    resources?: LitResourceAbilityRequest[]
+  ) => Promise<SessionSigsMap | undefined>
+): Promise<void> => {
+  devEnv.setUnavailable(LIT_TESTNET.MANZANO);
```

Comment:
🔪 ideally this would be replaced with wrapping the test definition function call itself in a conditional check, see above comment.

That being said, do we _really_ never run any test that uses this function on manzano -- and should it be disabled on `datil-test` if it was disabled on `manzano`?